purchases/src/main/kotlin/com/revenuecat/purchases/caching/DeviceCache.kt

```diff
@@ -61,6 +62,13 @@ internal class DeviceCache(
                     editor.remove(purchaserInfoCacheKey(it))
                 }
             }
+            // Subscriber attributes
+            .also { editor ->
+                getCachedAppUserID()?.let {
+                    editor.remove(subscriberAttributesCacheKey(it))
+                }
+            }
```

Comment:
code your comments > comment your code. Would something like this be viable? 
(pseudo)
```kotlin
fun clearSubscriberAttributes(editor) -> editor {
    editor.also { editor ->
                getCachedAppUserID()?.let {
                    editor.remove(subscriberAttributesCacheKey(it))
                }
            }
}
```